Date: Thu, 11 May 2021 
Subject: [PATCH] librmnetctl: RMNET_NETLINK_NEW_VND_WITH_NAME

diff --git a/opensource/dataservices/rmnetctl/src/librmnetctl.c b/opensource/dataservices/rmnetctl/src/librmnetctl.c
index 4cf2d27..c1da816 100644
--- a/opensource/dataservices/rmnetctl/src/librmnetctl.c
+++ b/opensource/dataservices/rmnetctl/src/librmnetctl.c
@@ -933,6 +933,7 @@
	if (prefix) {
- request.message_type =RMNET_NETLINK_NEW_VND_WITH_NAME;
+ request.message_type =RMNET_NETLINK_NEW_VND_WITH_PREFIX;
  str_len = strlcpy((char *)request.vnd.vnd_name,
	    prefix, RMNET_MAX_STR_LEN);
  if (_rmnetctl_check_len(str_len, error_code)
	      != RMNETCTL_SUCCESS)
