Date: Thu, 10 May 2021 
Subject: soong: Add flag for disable killing oneshot service

diff --git a/config/BoardConfigSoong.mk b/config/BoardConfigSoong.mk
index c0b0167..254c42c 100644
--- a/config/BoardConfigSoong.mk
+++ b/config/BoardConfigSoong.mk
@@ -30,6 +30,7 @@
 SOONG_CONFIG_lineageGlobalVars += \
     additional_gralloc_10_usage_bits \
     bootloader_message_offset \
+    disable_killing_oneshot_service \
     disable_postrender_cleanup \
     has_legacy_camera_hal1 \
     has_memfd_backport \
@@ -62,6 +63,7 @@
 endif
 
 # Soong bool variables
+SOONG_CONFIG_lineageGlobalVars_disable_killing_oneshot_service := $(TARGET_DISABLE_KILLING_ONESHOT_SERVICE)
 SOONG_CONFIG_lineageGlobalVars_disable_postrender_cleanup := $(TARGET_DISABLE_POSTRENDER_CLEANUP)
 SOONG_CONFIG_lineageGlobalVars_has_legacy_camera_hal1 := $(TARGET_HAS_LEGACY_CAMERA_HAL1)
 SOONG_CONFIG_lineageGlobalVars_has_memfd_backport := $(TARGET_HAS_MEMFD_BACKPORT)
