Date: Thu, 13 May 2021 
Subject: [PATCH] dataservices:use RMNET_NETLINK_NEW_VND_WITH_PREFIX *RMNET_NETLINK_NEW_VND_WITH_NAME is undeclared

diff --git a/rmnetctl/src/librmnetctl.c b/rmnetctl/src/librmnetctl.c
index d621f43..02ac794 100644
--- a/rmnetctl/src/librmnetctl.c
+++ b/rmnetctl/src/librmnetctl.c
@@ -931,7 +931,7 @@
 
 	memset(request.vnd.vnd_name, 0, RMNET_MAX_STR_LEN);
 		if (prefix) {
-			request.message_type =RMNET_NETLINK_NEW_VND_WITH_NAME;
+			request.message_type =RMNET_NETLINK_NEW_VND_WITH_PREFIX;
 			str_len = strlcpy((char *)request.vnd.vnd_name,
 					  prefix, RMNET_MAX_STR_LEN);
 			if (_rmnetctl_check_len(str_len, error_code)
