Date: Thu, 10 May 2021 
Subject: [PATCH] init: Add ability to disable killing oneshot service

diff --git a/init/Android.bp b/init/Android.bp
index e28d9f0..cc78233 100644
--- a/init/Android.bp
+++ b/init/Android.bp
@@ -170,6 +170,7 @@
         "init_defaults",
         "selinux_policy_version",
         "vendor_init_defaults",
+        "disable_killing_oneshot_service_defaults",
     ],
     srcs: init_common_sources + init_device_sources,
     whole_static_libs: [
