Date: Thu, 10 May 2021 
Subject: [PATCH] init: Add ability to disable killing oneshot service

diff --git a/init/service.cpp b/init/service.cpp
index d6f913c..94b7249 100644
--- a/init/service.cpp
+++ b/init/service.cpp
@@ -269,7 +269,10 @@
             // The new behavior in Android R is to kill these process groups in all cases.  The
             // 'true' parameter instructions KillProcessGroup() to report a warning message where it
             // detects a difference in behavior has occurred.
-            KillProcessGroup(SIGKILL, true);
+#ifdef DISABLE_KILLING_ONESHOT_SERVICE
+            if (!(flags_ & SVC_ONESHOT))
+#endif
+                KillProcessGroup(SIGKILL, true);
         }
     }
 
