Date: Thu, 28 May 2021 
Subject: [PATCH] rhine-common: rootdir: Introduce a service to start suntrold

diff --git a/rhine.mk b/rhine.mk
index 110c2fc..17d35aa 100644
--- a/rhine.mk
+++ b/rhine.mk
@@ -108,9 +108,9 @@
     $(COMMON_PATH)/rootdir/ueventd.qcom.rc:root/ueventd.qcom.rc
 
 
-PRODUCT_COPY_FILES += \
-    $(COMMON_PATH)/rootdir/system/vendor/bin/credmgrfirstboot.sh:$(TARGET_COPY_OUT_VENDOR)/bin/credmgrfirstboot.sh
-
+PRODUCT_PACKAGES += \
+    credmgrfirstboot.sh \
+    suntrold.sh
 
 # Connectivity
 PRODUCT_PACKAGES += \
