Date: Thu, 10 May 2021 
Subject: [PATCH] soong: Add ability to enable product_shipping_api_level variant.

diff --git a/android/variable.go b/android/variable.go
index 4cf2d27..c1da816 100644
--- a/android/variable.go
+++ b/android/variable.go
@@ -56,6 +56,10 @@
 			Exclude_static_libs []string `android:"arch_variant"`
 		} `android:"arch_variant"`
 
+		Product_shipping_api_level struct {
+			Cflags  []string
+		}
+
 		Safestack struct {
 			Cflags []string `android:"arch_variant"`
 		} `android:"arch_variant"`
@@ -215,6 +219,7 @@
 	Unbundled_build_sdks_from_source *bool `json:",omitempty"`
 	Malloc_not_svelte                *bool `json:",omitempty"`
 	Malloc_not_svelte_libc32         *bool `json:",omitempty"`
+	Product_shipping_api_level       *int  `json:",omitempty"`
 	Safestack                        *bool `json:",omitempty"`
 	HostStaticBinaries               *bool `json:",omitempty"`
 	Binder32bit                      *bool `json:",omitempty"`
