Date: Thu, 10 May 2021 
Subject: [PATCH] soong: Add flag for disable killing oneshot service

diff --git a/build/soong/Android.bp b/build/soong/Android.bp
index ab30e07..9be13b6 100644
--- a/build/soong/Android.bp
+++ b/build/soong/Android.bp
@@ -49,6 +49,23 @@
     recovery_available: true,
 }
 
+soong_config_module_type {
+    name: "disable_killing_oneshot_service",
+    module_type: "cc_defaults",
+    config_namespace: "lineageGlobalVars",
+    bool_variables: ["disable_killing_oneshot_service"],
+    properties: ["cppflags"],
+}
+
+disable_killing_oneshot_service {
+    name: "disable_killing_oneshot_service_defaults",
+    soong_config_variables: {
+        disable_killing_oneshot_service: {
+            cppflags: ["-DDISABLE_KILLING_ONESHOT_SERVICE"],
+        },
+    },
+}
+
 // Target platform agnostic config modules
 soong_config_module_type {
     name: "disable_postrender_cleanup",
