Date: Thu, 10 May 2021 
Subject: [PATCH] soong: Add ability to enable product_shipping_api_level variant.

diff --git a/core/soong_config.mk b/core/soong_config.mk
index 7c6410a..cadb0e2 100644
--- a/core/soong_config.mk
+++ b/core/soong_config.mk
@@ -142,6 +142,7 @@
 $(call add_json_bool, Malloc_not_svelte_libc32,          $(if $(MALLOC_SVELTE_FOR_LIBC32),\
                                                             $(call invert_bool,$(filter true,$(MALLOC_SVELTE_FOR_LIBC32))),\
                                                             $(call invert_bool,$(filter true,$(MALLOC_SVELTE)))))
+$(call add_json_val,  Product_shipping_api_level,        $(PRODUCT_SHIPPING_API_LEVEL))
 $(call add_json_str,  Override_rs_driver,                $(OVERRIDE_RS_DRIVER))
 
 $(call add_json_bool, UncompressPrivAppDex,              $(call invert_bool,$(filter true,$(DONT_UNCOMPRESS_PRIV_APPS_DEXS))))
